.gitignore

```diff
@@ -0,0 +1,5 @@
+.idea
```

Comment:
so its only the *.iml and the out/* ?

is there a problem? because i was on the most recent version i think...